research/object_detection/dataset_tools/seq_example_util_test.py

```diff
@@ -122,7 +122,8 @@ def test_make_labeled_example(self):
     bboxes = [
         np.array([[0., 0., 0., 0.],
                   [0., 0., 1., 1.]], dtype=np.float32),
-        np.zeros([0, 4], dtype=np.float32)
+        np.zeros([0, 4], dtype=np.float32),
+        np.array([], dtype=np.float32)
```

Comment:
I am not sure I understand this change. There are only 2 frames on line 107, why do we have 3 sets of boxes ? 
seq_example_util.py line 151. Can you explain that ?